collector/src/main/java/com/navercorp/pinpoint/collector/receiver/grpc/GrpcServerResponse.java

```diff
@@ -26,15 +28,28 @@
  */
 public class GrpcServerResponse<T> implements ServerResponse<T> {
     private final StreamObserver<T> responseObserver;
+    private final boolean retryFriendlyResponse;
 
     public GrpcServerResponse(StreamObserver<T> responseObserver) {
+        this(responseObserver, false);
+    }
+
+    public GrpcServerResponse(StreamObserver<T> responseObserver, boolean retryFriendlyResponse) {
         this.responseObserver = Objects.requireNonNull(responseObserver, "responseObserver");
+        this.retryFriendlyResponse = retryFriendlyResponse;
     }
 
     @Override
     public void write(final T message) {
         Objects.requireNonNull(message, "message");
-
+        if (retryFriendlyResponse) {
```

Comment:
Could you check create new GrpcServerResponse type for the RetryFriendlyResponse?